web/modules/custom/dpl_event/src/EventSubscriber/EventSeriesRedirect.php

```diff
@@ -16,6 +18,14 @@
  */
 class EventSeriesRedirect implements EventSubscriberInterface {
 
+  /**
+   * {@inheritdoc}
+   */
+  public function __construct(
+    private EntityTypeManagerInterface $entityTypeManager,
+    private ReoccurringDateFormatter $reoccurringDateFormatter,
```

Comment:
Consider making these properties camel case.

I know this can be a mixed bag but in general [the Drupal code guidelines](https://www.drupal.org/docs/develop/standards/php/php-coding-standards#s-functions-and-variables) says:

> Be consistent; do not mix camelCase and snake_case variable naming inside a file.

Reply:
The reason I'm making them camelcase is because it's autowiring (is that the name?) - if i make it snake case, it'll end up being used as `$this->entity_type_manager->yada..` which seems inconsistent with how we normally do it

Reply:
I think you are referring to dependency injection. As far as I know this is not related to casing. The service does not use autowiring at the moment as you defined the arguments in `service.yml`.